fluid/PaddleCV/image_classification/dist_train/dist_train.py

```diff
@@ -221,8 +271,21 @@ def train_parallel(args):
         append_bn_repeat_init_op(train_prog, startup_prog, args.multi_batch_repeat)
     startup_exe.run(startup_prog)
 
+    np_tensors = {}
+    np_tensors["img_mean"] = np.array([0.485, 0.456, 0.406]).astype("float16" if args.fp16 else "float32").reshape((3, 1, 1))
+    np_tensors["img_std"] = np.array([0.229, 0.224, 0.225]).astype("float16" if args.fp16 else "float32").reshape((3, 1, 1))
+    np_tensors["img_scale"] = np.array([255.0]).astype("float16" if args.fp16 else "float32")
+    for vname, np_tensor in np_tensors.items():
+        var = fluid.global_scope().find_var(vname)
+        if args.fp16:
+            var.get_tensor().set(np_tensor.view(np.uint16), place)
+        else:
+            var.get_tensor().set(np_tensor, place)
+
+
     strategy = fluid.ExecutionStrategy()
     strategy.num_threads = args.num_threads
+    strategy.num_iteration_per_drop_scope = 30
```

Comment:
this option seems not friendly to users. Do we have any other configuration method for this?

Reply:
Done.